master/static/srv/notebook-entrypoint.sh

```diff
@@ -18,6 +18,30 @@ set -x
 test -f "${STARTUP_HOOK}" && source "${STARTUP_HOOK}"
 set +x
 
+if [ "$PACHD_ADDRESS" != "" ] && [ "$SKIP_PACHYDERM_INSTALL" != "true" ]; then
+    version=$(echo $(curl -s -X POST -H "Content-Type: application/json" pachyderm-proxy:80/api/versionpb_v2.API/GetVersion) | sed -n 's/.*"major":\([0-9]*\), "minor":\([0-9]*\), "micro":\([0-9]*\).*/\1.\2.\3/p')
```

Comment:
That `sed` is hard for me to parse and verify -- I don't know how the payload return from that endpoint looks. But it's complicated enough that I feel I should be able to read and understand it. How should I review it?

Related, how should this line be tested? Let's say `/api/versionpb_v2.API/GetVersion` changes. How will our testing alert us that something is broken?

I think I'd like to see at least this small part tested somewhere in `e2e_tests`. We've got precedence for testing endpoints with `./ship-logs.sh` and `./e2e_tests/tests/cluster/test_ship_logs.py`. But... I don't know. This also seems complicated, because how will the CI version Pachyderm?

What are others' thoughts?